tests/support/mocks/common/party.ts

```diff
@@ -5,7 +5,7 @@ import has from '@dojo/core/has';
 // this case, `node_modules/@dojo/loader/loader.min.js'. Is there a better, less hacky way to handle this?
 const hasHostNode = has('host-node');
 const pathSeparator = hasHostNode ? require('path').sep : '/';
-const basePath = hasHostNode ? `..${pathSeparator}_build${pathSeparator}` : '';
+const basePath = hasHostNode ? `${__dirname}${pathSeparator}..${pathSeparator}..${pathSeparator}..${pathSeparator}..${pathSeparator}` : `_/build${pathSeparator}`;
```

Comment:
This would be much better as `path.resolve(__dirname, '../../../../')` (and the path separator will be converted automatically).

Reply:
Agreed. updated!